signalfx-org-metrics/metrics.yaml

```diff
@@ -1371,21 +1369,66 @@ sf.org.subscription.activeTimeSeries:
   brief: Maximum number of active MTS
   description: |
     Maximum number of active metric time series (MTS).
-
+        
         * Data resolution: 15 minutes
   metric_type: gauge
   title: sf.org.subscription.activeTimeSeries
 
+sf.org.subscription.containers:
```

Comment:
i think all the sf.org.subscription things are what was purchased not what the limit is.